gwak/train/configs/bbh.yaml

```diff
@@ -19,17 +21,19 @@ model:
     num_timesteps: 200
     bottleneck: 8
 data:
-  class_path: dataloader.SignalDataloader
+  class_path: dataloader.BBHDataloader
   init_args:
-    data_dir: /home/ethan.marx/aframe/data/train/background
+    data_dir: /home/hongyin.chen/Data/GWAK_data/gwak2_background
     sample_rate: 2048
     kernel_length: 0.09765625
     psd_length: 64
     fduration: 1
     fftlength: 2
-    batch_size: 128
-    batches_per_epoch: 128
+    batch_size: 8
+    batches_per_epoch: 8
     num_workers: 5
-    prior: gwak.data.prior.BBHPrior
+    prior: gwak.data.prior.LAL_BBHPrior
     waveform:
-      class_path: ml4gw.waveforms.IMRPhenomPv2
\ No newline at end of file
+      class_path: ml4gw.waveforms.IMRPhenomPv2
+    ringdown_duration: 0.9
+    data_saving_file: /home/hongyin.chen/Data/gwak/bbh.h5
```

Comment:
we should think of a way to use a globally defined path here, instead of one with a specific username